JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1862,6 +1901,7 @@ class AppInsightsTests extends TestClass {
                 var snippet = this.getAppInsightsSnippet();
                 snippet.disableAjaxTracking = false;
                 snippet.disableCorrelationHeaders = false;
+                snippet.enableCorsCorrelation = true;
```

Comment:
this should be tested with false as default?

Reply:
this is the test for excluded domains that are honored only for cross-domain. i have different tests for  all combinations of those flags and every other test has enableCorsCorrelation  as false by the default.